IS7024FinalProject/DTOs/ParkWhizDTOs.cs

```diff
@@ -57,59 +60,50 @@ public class ParkWhizQuote
         public string Type { get; set; } = string.Empty;
 
         [JsonPropertyName("distance")]
-        public ParkWhizDistance Distance { get; set; } = new ParkWhizDistance();
+        public ParkWhizDistance Distance { get; set; } = new();
 
         [JsonPropertyName("purchase_options")]
-        public List<PurchaseOption> PurchaseOptions { get; set; } = new List<PurchaseOption>();
+        public List<PurchaseOption> PurchaseOptions { get; set; } = new();
 
-        // New field to capture the nested embedded data
         [JsonPropertyName("_embedded")]
-        public ParkWhizEmbedded Embedded { get; set; } = new ParkWhizEmbedded();
+        public ParkWhizEmbedded Embedded { get; set; } = new();
 
         // --- Calculated Properties for Easy Access and Display ---
-        
+
         public string DisplayLocationName => Embedded.Location?.Name ?? string.Empty;
-        
-        // Construct the full address from the nested fields
+
         public string DisplayStreetAddress
         {
             get
             {
                 if (Embedded.Location == null) return string.Empty;
 
-                // Concatenate Address1, City, and State for a complete address string
-                var parts = new List<string> { Embedded.Location.Address1, Embedded.Location.City };
-                
-                // Filter out empty parts before joining
+                var parts = new List<string>
+                {
+                    Embedded.Location.Address1,
+                    Embedded.Location.City,
+                    Embedded.Location.State
+                };
+
                 parts.RemoveAll(string.IsNullOrWhiteSpace);
-                
                 return string.Join(", ", parts);
             }
         }
 
-        // NEW: Property to grab the URL for the first photo's "gallery" size
-        public string DisplayImageUrl
-        {
-            get
-            {
-                // Try to get the URL from the first photo
-                var imageUrl = Embedded.Location?.Photos.FirstOrDefault()?.Sizes?.Gallery?.Url;
-
-                // Return the URL or a styled placeholder if no image is available
-                // Using a standard placeholder image URL as a string fallback (400x300 for card)
-                return imageUrl ?? "https://placehold.co/400x300/e9ecef/495057?text=Parking+Image+N%2FA";
-            }
-        }
+        public string DisplayImageUrl =>
+            Embedded.Location?.Photos.FirstOrDefault()?.Sizes?.Gallery?.Url
+            ?? "https://placehold.co/400x300/e9ecef/495057?text=Parking+Image+N%2FA";
```

Comment:
These two are doing the same thing, but the suggestion consolidates to fewer lines.
There are good reasons for spreading something out to multiple lines:
- Documentation/readability
- If an error happens, you get a line number.  If that line number is doing one thing, the source of the error is obvious.  If that line is doing multiple things, it is harder to identify which of those things caused the error.

I'd probably stick with the original in this case.